lib/dradis/plugins/veracode/field_processor.rb

```diff
@@ -2,25 +2,30 @@ module Dradis
   module Plugins
     module Veracode
       class FieldProcessor < Dradis::Plugins::Upload::FieldProcessor
-        def post_initialize(args={})
+        def post_initialize(args = {})
```

Comment:
`attr_accessor :record` ?

Reply:
I don't think we need this? The other field processors don't have accessors too